ServerCore/Pages/Resources/pd2021.2203/HomePartial.cshtml

```diff
@@ -0,0 +1,60 @@
+﻿@{
+    var eventPage = Model as ServerCore.ModelBases.EventSpecificPageModel;
+    bool IsRegistered = false;
+    if (eventPage != null)
+    {
+        IsRegistered = await eventPage.IsRegisteredUser();
+    }
+}
+
+<div class="container" id="pd2021-container">
+    <h2 class="pd2021-title">
+        <span>Intern Puzzleday 2021 ver. 2022</span>
+    </h2>
+    <div>
+        <img class="divider-img" src="~/images/divider.png">
+    </div>
+
+    <div class="pd2021-container-root">
+        <div class="pd2021-container-card">
+            <h3>WELCOME TO PUZZLEDAY</h3>
+            <p>We&rsquo;re honored to have you join us in what we&rsquo;re sure will be an event for the history books: Intern Puzzleday 2021: 2022 Edition! Our authors have written enough puzzles to keep your eyes glued to the pages for two days this March. And the best news: you don&rsquo;t need to worry about late fees!</p>
+
+            <p>This year&rsquo;s event will take place over two days: <b>Thursday, March 24<sup>th</sup></b> and <b>Friday, March 25<sup>th</sup></b>. On both of those days, you&rsquo;ll have 4 hours from <b>1pm-5pm UTC</b> to solve puzzles, and both days of solving contribute to the overall event.  Some may also choose to keep solving after the end of the first day, but doing so is not required.</p>
+
+            <p>Take a look at our <b><a asp-page="/EventSpecific/FAQ">FAQ</a></b> and <b><a asp-page="/EventSpecific/Rules">Rules</a></b> if you have any questions.</p>
+
+            <p>We invite participation by FTEs, friends, and families of all abilities. If you require an accommodation,  please reach out to us at <a href="mailto:puzzhunt@microsoft.com">puzzhunt@microsoft.com</a>.</p>
```

Comment:
FTE? You sure about that?

Reply:
Aha, I KNEW I'd forget one!  Fixing in next commit.